lib/metasploit_payloads/mettle.rb

```diff
@@ -40,7 +40,11 @@ def to_binary(format=:process_image)
     def generate_argv
       cmd_line = 'mettle '
       @config.each do |opt, val|
-        cmd_line << "-#{short_opt(opt)} \"#{val}\" "
+        if val.is_a? String
+          cmd_line << "-#{short_opt(opt)}\"#{val}\" "
+        else
+          cmd_line << "-#{short_opt(opt)}#{val.to_s} "
```

Comment:
String interpolation always calls `#to_s` in Ruby.
```
irb(main):001:0> "#{9}"
=> "9"
```

Reply:
Likely something I am misunderstanding; I recall seeing a few errors generated by rubocop to "prefer to_s over string interpolation".

Reply:
When it's just the one thing that you want to make into a string (like my example), correct. Just call `#to_s` and don't mess with interpolation. But when you have to interpolate anyway, there's no need to add redundant `#to_s`s to anything.

Reply:
In other words, `"#{foo}"` calls .to_s implicitly. `"#{foo.to_s}"` is the same as `foo.to_s.to_s`. rubocop will not complain if you have other stuff in your string.